how-to/markdown.md

```diff
@@ -0,0 +1,9 @@
+# How to include a Markdown file
```

Comment:
Do we really want to include gibberish in this published file?

Reply:
We have that all over the starter pack at the moment - but we should split it up, see https://github.com/canonical/sphinx-docs-starter-pack/issues/25

Reply:
:astonished: oh